package.json

```diff
@@ -23,11 +23,10 @@
     "swiftlint": "node-swiftlint",
     "prepack": "yarn run build",
     "postinstall": "node scripts/check-siblings.js",
-    "bump:v5": "cd example/ionic-angular-v5 && yalc add @sentry/capacitor && yarn install && ionic build && npx cap sync",
-    "bump:v3": "cd example/ionic-angular && yalc add @sentry/capacitor && yarn install && ionic build && npx cap sync",
-    "bump:v2": "cd example/ionic-angular-v2 && yalc add @sentry/capacitor && yarn install && ionic build && npx cap sync",
-    "bump:sample-vue": "cd example/ionic-vue3 && yalc add @sentry/capacitor && yarn install && ionic build && npx cap sync",
```

Comment:
Why is the `sample-vue` removed?

Reply:
Nice catch, something went wrong on the merge from the main branch